sphinx_pyscript/mini-coi.js

```diff
@@ -0,0 +1,28 @@
+/*! coi-serviceworker v0.1.7 - Guido Zuidhof and contributors, licensed under MIT */
```

Comment:
Add a link here to https://docs.pyscript.net/2024.5.1/user-guide/workers/, to explain the purpose of this script

Reply:
Good point - added an explanatory comment and link.